configuration/concepts/orders_AllOrderable.csv

```diff
@@ -0,0 +1,30 @@
+81e95bce-3f10-11e4-adec-0800271c1b75,,,Radiology Orders,Radiology Orders,Radiology,Radiologie,,,ConvSet,N/A,,,5459f481-bc1f-4719-8972-ab58cb98a05b,,,,,,,,
+aa6fbb7c-e559-4149-893e-9d3f6a2db01c,,,All Surgery Orders,All Surgery Orders,Surgery,Chirurgie,,,ConvSet,N/A,,,9b5abdcf-cfda-4c47-b26c-b0434d5a82b8;03656494-9a4c-4852-ac55-692be25a5414,,,,,,,,
+6b3a8b87-9eaa-4202-916a-d785326a7a2d,,,All ENT Orders,All ENT Orders,ENT,ORL,,,ConvSet,N/A,,,8350f3b0-321b-41ac-b2fb-fbd916774fcf,,,,,,,,
+11675455-21d6-46f3-a333-e996f9224cc2,,,All Urology Orders,All Urology Orders,Urology,Urologie,,,ConvSet,N/A,,,0c663543-ce77-4f75-8e57-0e1bf9fbda75,,,,,,,,
+d81ccd49-d60c-4661-a348-f7785e561229,,,All Wounds Orders,All Wounds Orders,Wounds,Plaies et Lacerations        ,,,ConvSet,N/A,,,cb60d618-b444-4259-b88c-1fdfbf56d9b4;6c163245-1076-4b34-889c-f11be78cb9d5;73610882-cdb4-4667-a60d-dbb25430434a,,,,,,,,
+e6875a30-957e-41c6-b041-ad3fd68d4fc3,,,All Procedure Orders,All Procedure Orders,Procedure,Procedures,,,ConvSet,N/A,,,24f8bf24-ffa3-4eb5-878d-e84859fa8e33,,,,,,,,
+a92121cf-69c7-418f-8684-4ab15475ee61,,,All Orthopedics and Traumatology Orders,All Orthopedics and Traumatology Orders,Orthopedics and Traumatology,Orthopedie/Traumatologie,,,ConvSet,N/A,,,9feb7178-741a-49e5-8888-2399bbf0217b,,,,,,,,
+bd46a00b-9b09-454a-ae32-58d27b2a6774,,,All Ophtalmology Orders,All Ophtalmology Orders,Ophtalmology,Ophtalmologie   ,,,ConvSet,N/A,,,a80d0d20-6fa4-4651-86ce-4eb9f43318a8,,,,,,,,
+259e4136-16fe-468a-aae7-683f8775b9f3,,,All Maxillofacial Orders,All Maxillofacial Orders,Maxillofacial,Maxillo-Facial,,,ConvSet,N/A,,,ddfe089d-bb0b-4a3d-bed6-33f69d9d92fd,,,,,,,,
+089d0ffc-f40b-4e48-b65a-582214669776,,,All Obsterics and Gynecology Orders,All Obsterics and Gynecology Orders,Obsterics and Gynecology,Obstetrie/Gyneco,,,ConvSet,N/A,,,21f1bc59-b811-4f50-a5f0-bcb1c354bad0,,,,,,,,
+6c1c0bcb-17ba-4a9f-9824-40e2fff787cd,,,All Materials Orders,All Materials Orders,Materials,Materiels,,,ConvSet,N/A,,,ae77ccee-f61e-4226-95fa-a494db8804fe,,,,,,,,
+,,,All Orderables,All Orderables,All Orderables,All Orderables,,,ConvSet,N/A,,,aa6fbb7c-e559-4149-893e-9d3f6a2db01c;6b3a8b87-9eaa-4202-916a-d785326a7a2d;11675455-21d6-46f3-a333-e996f9224cc2;d81ccd49-d60c-4661-a348-f7785e561229;e6875a30-957e-41c6-b041-ad3fd68d4fc3;a92121cf-69c7-418f-8684-4ab15475ee61;bd46a00b-9b09-454a-ae32-58d27b2a6774;259e4136-16fe-468a-aae7-683f8775b9f3;089d0ffc-f40b-4e48-b65a-582214669776;6c1c0bcb-17ba-4a9f-9824-40e2fff787cd;4967ef60-498b-4aa4-9c07-782073c364dd;81e95bce-3f10-11e4-adec-0800271c1b75;8160a011-3f10-11e4-adec-0800271c1b75,,,,,,,,
```

Comment:
There is a reference here to concept `4967ef60-498b-4aa4-9c07-782073c364dd`. I can't find this one anywhere. Which concept is it?
Because it is missing, it prevents the _All Orderables_ to be created.

Reply:
yes, Thanks @rbuisson . this one is unavailable concept Uuid was used only locally when switching to the new list of radiology and lab orders. it should be deleted!